apprise/plugins/NotifySlack.py

```diff
@@ -359,87 +373,129 @@ def send(self, body, title='', notify_type=NotifyType.INFO, attach=None,
         # error tracking (used for function return)
         has_error = False
 
-        # Perform Formatting
-        title = self._re_formatting_rules.sub(  # pragma: no branch
-            lambda x: self._re_formatting_map[x.group()], title,
-        )
-        # Only for NONE markdown, otherwise eg links wont work
```

Comment:
@caronc The HTML enconding breaks at least links in Markdown. Any particular reason you flipped the logic here ?

Reply:
![image](https://user-images.githubusercontent.com/850374/141645299-5a71ad15-5a62-4c98-84dc-50bd974b9357.png)
Yeah; i was able to successfully escape the title everywhere and tested it (working great) except a specific scenario; so i just handled it for that case.